src/recommendations/RecommendationsPage.jsx

```diff
@@ -46,8 +64,8 @@ const RecommendationsPage = () => {
 
   const handleSkipRecommendationPage = () => {
     window.history.replaceState(location.state, null, '');
-    if (registrationResponse) {
-      window.location.href = registrationResponse.redirectUrl;
+    if (redirectUrl) {
+      window.location.href = redirectUrl;
```

Comment:
something to discuss before merging: 

I think it's a security risk to redirect to `redirectUrl` coming in query params without verifying if its safe or not.